src/main/include/commands/intake/ResetIntakeEncoderCommand.h

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+#pragma once
+
+#include <frc2/command/CommandBase.h>
+#include <frc2/command/CommandHelper.h>
+
+#include "Utils.h"
+#include "subsystems/Intake.h"
+
+/**
+ *
+ */
+class ResetIntakeEncoderCommand
+    : public frc2::CommandHelper<frc2::CommandBase, ResetIntakeEncoderCommand> {
+public:
+    /**
+     * Creates a new ResetIntakeEncoderCommand.
+     *
+     * @param subsystem The subsystem used by this command.
+     */
+    explicit ResetIntakeEncoderCommand(std::shared_ptr<Intake> intake);
```

Comment:
what's the `explicit` for?

Reply:
It's to prevent implicit conversions between the arguments and the command. This is simply the default way WPILib set up the example command, and it's a fine practice so I just copy pasted it. You can read more about it [here](https://stackoverflow.com/questions/121162/what-does-the-explicit-keyword-mean)